users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -176,6 +178,29 @@ def get_masked():
             total_time_masks_max_frames // time_mask_max_size,
         )
         actual_freq_mask_max_num = tf.minimum(freq_mask_max_num, total_freq_masks_max_size // freq_mask_max_size)
+         
+        # Check if limits where hit and which one
+        time_lower_limit_hit = tf.equal(actual_time_mask_max_num, max_time_num_seq_len)
+        time_upper_limit_hit = tf.equal(actual_time_mask_max_num, total_time_masks_max_frames // time_mask_max_size)
+        freq_limit_hit = tf.equal(actual_freq_mask_max_num, total_freq_masks_max_size // freq_mask_max_size)
+
+        enable_logging = tf.convert_to_tensor(config["enable_logging"], dtype=tf.bool)
+
+        def logging_ops():
+
+            with tf.control_dependencies([
+                tf.print(
+                "Specaug Log: ",
+                current_epoch,
+                actual_time_mask_max_num,
+                actual_freq_mask_max_num,
```

Comment:
What about `time_mask_max_size` and same for frequency?

Reply:
time_mask_max_size is fixed and does not need to be logged.